libs/application/template-api-modules/src/lib/types/index.ts

```diff
@@ -80,3 +84,11 @@ export type AttachmentEmailTemplateGenerator = (
   fileContent: string,
   email: string,
 ) => Message
+
+export interface ExternalDataNationalRegistry {
+  data: NationalRegistryUser
+}
+
+export interface ExternalDataJuristiction {
```

Comment:
Since `JuristictionProvider` is only a part of the `driving-license` template I don't think it should live in a generic type file for all applications.

Reply:
agree.. will fix